### PR TITLE
Python3 support

### PR DESCRIPTION
**Pull request by themperek**

### CODE REVIEW

File: tests/test_SimGpio.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
         offset  : 7
       - name    : IN
         size    : 8
-        offset  : 14
```
[the remark this answers is not in the document]
Comment (themperek) [reply tests/test_SimGpio.py]: Yes.

File: basil/dut.py [hunk 2]
```diff
@@ -300,11 +304,11 @@ def get_modules(self, type_name):
         return modules
 
     def __iter__(self):
-        for item in self._registers.itervalues():
```
[the remark this answers is not in the document]
Comment (themperek) [reply basil/dut.py]: Not much.

**Comment by themperek** [comment]
Should we squash all or rebase?

**Comment by laborleben** [comment]
If you squash you'd still need to rebase

**Comment by laborleben** [comment]
But you can do everything at once: git rebase -i development

**Comment by DavidLP** [comment]
I am in favor of rebase + merge and no squash. Some int / float divisions look fishy how it was done in the past ...

**Comment by laborleben** [comment]
Can you point out the suspicious places?

**Comment by DavidLP** [comment]
For example: https://github.com/SiLab-Bonn/basil/blob/development/basil/HL/MIO_PLL.py#L127
The result depends on if `self.fref` is an `int` or a `float`.

**Comment by laborleben** [comment]
I agree, but this is the only file affected to my mind. You remember the discussion about floating point precision (exact comparison between values)? :-)

This file must be revisited as a whole.

**Comment by DavidLP** [comment]
OK, good. Then we do not have to change this. Shall I press `Rebase and merge`?

**Comment by laborleben** [comment]
Yes, just do it :-)

**Comment by laborleben** [comment]
Can we also remove the py3k branch afterwards?

**Comment by DavidLP** [comment]
> Can we also remove the py3k branch afterwards?

Done

**Pull request merged**